drivers/espi/espi_mchp_xec.c

```diff
@@ -0,0 +1,822 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <soc.h>
+#include <errno.h>
+#include <espi.h>
+#include "espi_utils.h"
+
+#define M2S             1ul
+#define S2M             0ul
+#define SMVW_ID_MAX     10ul
+
+#define SRC_ID0         0
+#define SRC_ID1         1
+#define SRC_ID2         2
+#define SRC_ID3         3
+#define SRC_ID_MAX      4
+
+struct espi_isr {
+	u32_t girq_bit;
+	void (*the_isr)(struct device *dev);
```

Comment:
it is up to you, but I would name it "isr" instead.

Reply:
+1